### PR TITLE
Default disable tracing some ai events

### PR DESCRIPTION
**Pull request by gavinfish**
Need to decide what ai events should be filtered here.

### CODE REVIEW

File: azure-commons-plugin/pom.xml [hunk 1]
```diff
@@ -15,6 +15,12 @@
     <description>Common APIs for Azure related Jenkins plugins</description>
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Azure+Commons+Plugin</url>
 
+    <properties>
+        <enableAzureApiTrace>false</enableAzureApiTrace>
```
Comment (yungezz) [inline azure-commons-plugin/pom.xml:19]: what if azure api call failed? will there be trace?

File: azure-commons-plugin/pom.xml [hunk 2]
```diff
@@ -15,6 +15,12 @@
...
+        <enablePageDecorator>false</enablePageDecorator>
```
Comment (yungezz) [inline azure-commons-plugin/pom.xml:20]: what's this?

Comment (gavinfish) [reply azure-commons-plugin/pom.xml:20]: When one of the pages in Jenkins being rendering, the page decorator can be notified and decide whether to send a ping event to our AI.

File: azure-commons-plugin/pom.xml (hunk 1 above)
Comment (gavinfish) [reply azure-commons-plugin/pom.xml:19]: I can still trace the failed api call if necessary. Our BI does not track these data for now.

**Review by yuwzho — APPROVED** [review]

**Pull request merged**